packages/core/src/Provider/CardanoProvider.ts

```diff
@@ -4,6 +4,7 @@ import { Schema as Cardano } from '@cardano-ogmios/client';
 type Tx = { hash: Cardano.Hash16 } & Cardano.Tx;
 
 export interface CardanoProvider {
+  /** @param signedTransaction signed and serialized cbor */
```

Comment:
Also adding documentation here for more detail on the param since the type does help. We should probably improve the type directly, but it's not critical at this point.